portfolio/src/main/webapp/calendarApi.js

```diff
@@ -0,0 +1,283 @@
+var GoogleAuth;
+
+// Scopes for API access to Google Calendar
+const SCOPE_READ_ONLY = 'https://www.googleapis.com/auth/calendar.readonly';
+const SCOPE_READ_WRITE = 'https://www.googleapis.com/auth/calendar';
+
+// For the discovery document for Google Calendar API.
+const DISCOVERY_URL =
+    'https://www.googleapis.com/discovery/v1/apis/calendar/v3/rest';
+
+const CLIENT_ID =
+    '499747085593-hvi6n4kdrbbfvcuo1c9a9tu9oaf62cr2.apps.googleusercontent.com';
+
+// Object for all error codes
+const ERROR_CODES =
```

Comment:
nit: I think the line break before { is not necessary.

ex:
const ERROR_CODES = {
  ...
};

Note the semicolon at the end.

Reply:
Updated.